scss/general.scss

```diff
@@ -0,0 +1,8 @@
+@import "./_variables.scss";
+
+@import "./sections/_header.scss";
+@import "./sections/_hero.scss";
+@import "./sections/_offer.scss";
+@import "./sections/_informations.scss";
+@import "./sections/contact";
+@import "./_mediaqueries.scss";
```

Comment:
[REQ]: Please remove this file it's bad approach to keep all media in one file. Instead do is in this way create media in each section dedicated file.